netlify.toml

```diff
@@ -1,4 +1,7 @@
+[build.environment]
+  NPM_FLAGS = "--version"
+
 [build]
   base = "www"
   publish = "_site"
-  command = "npm run build"
\ No newline at end of file
+  command = "npx pnpm i --store=node_modules/.pnpm-store && npm run build"
```

Comment:
_very_ clever solution to avoid needing `pnpm build`. If it works, this is cool!

Reply:
It took me a bit to realize how to use pnpm with netlify, I followed this thread https://answers.netlify.com/t/using-pnpm-and-pnpm-workspaces/2759/16